src/main/webapp/onelive/index.html

```diff
@@ -277,13 +285,75 @@ <h5 class="title">
     </div>
 </script>
 
+<script id="upcoming-events-template" type="text/html">
+    <div class="col-md-10 offset-1">
+        <h3>Upcoming</h3>
+    </div>
+
+    {{#data}}
+    <div class="col-md-10 offset-1">
+        <a href="{{facebookLink}}" target="_blank">
+            <div class="card card-event shadow shadow-lg--hover mb-1 text-body">
+                <div class="card-body">
+                    <span class="h5 mr-1 card-event-date">{{date}}</span>
+                    <span class="small text-muted mr-4 text-uppercase">{{month}}, {{day}}</span>
+                    <span class="mr-5"> {{startTime}} - {{endTime}}</span>
+                    <span class="font-weight-bold ml-1">{{summary}}</span>
+                </div>
+            </div>
+        </a>
+    </div>
+    {{/data}}
+</script>
+
 <script>
     $(function () {
         loadNavAndFooter('/assets/content/static');  //relative path to content directory
+        loadUpcomingEvents(); //load upcoming events
         loadProfiles(); // load profiles
         loadFaqComponent();
     });
 
+    //function to load upcoming events
+    function loadUpcomingEvents() {
+        $.ajax({
+            type: 'GET',
+            url: 'http://localhost:8080/calendar/onelive',
```

Comment:
localhost:8080?
Isn't this service up and running?

Reply:
Oh yes. I forgot to change it. I'll update